File: src/main/java/entities/gestionStore/Livraison.java
package entities.gestionStore;

public class Livraison {

    private int idLivraison;
    private int idFacture;
    private int idClient;
    private String Lieu;
    private String etat;

    public int getIdLivraison() {
        return idLivraison;
    }

    public void setIdLivraison(int idLivraison) {
        this.idLivraison = idLivraison;
    }

    public int getIdFacture() {
        return idFacture;
    }

    public void setIdFacture(int idFacture) {
        this.idFacture = idFacture;
    }

    public int getIdClient() {
        return idClient;
    }

    public void setIdClient(int idClient) {
        this.idClient = idClient;
    }

    public String getLieu() {
        return Lieu;
    }

    public void setLieu(String lieu) {
        Lieu = lieu;
    }

    public String getEtat() {
        return etat;
    }

    public void setEtat(String etat) {
        this.etat = etat;
    }

    public Livraison() {
    }

    public Livraison(int idFacture, int idClient, String lieu, String etat) {
        this.idFacture = idFacture;
        this.idClient = idClient;
        Lieu = lieu;
        this.etat = etat;
    }

    public Livraison(int idLivraison, int idFacture, int idClient, String lieu, String etat) {
        this.idLivraison = idLivraison;
        this.idFacture = idFacture;
        this.idClient = idClient;
        Lieu = lieu;
        this.etat = etat;
    }

    @Override
    public String toString() {
        return "Livraison{" +
                "idLivraison=" + idLivraison +
                ", idFacture=" + idFacture +
                ", idClient=" + idClient +
                ", Lieu='" + Lieu + '\'' +
                ", etat='" + etat + '\'' +
                '}';
    }
}

File: webapp/src/GymPlus/src/Entity/Livraison.php
<?php

namespace App\Entity;

use App\Repository\LivraisonRepository;
use Doctrine\DBAL\Types\Types;
use Doctrine\ORM\Mapping as ORM;

#[ORM\Entity(repositoryClass: LivraisonRepository::class)]
class Livraison
{
    #[ORM\Id]
    #[ORM\GeneratedValue]
    #[ORM\Column]
    private $idlivraison;

    #[ORM\Column]

    private ?int $idfacture;

    #[ORM\Column]
    private ?int $idclient;

    #[ORM\Column]
    private ?string $lieu;

    #[ORM\Column]
    private ?string $etat;

    public function getIdlivraison(): ?int
    {
        return $this->idlivraison;
    }

    public function getIdfacture(): ?int
    {
        return $this->idfacture;
    }

    public function setIdfacture(?int $idfacture): static
    {
        $this->idfacture = $idfacture;

        return $this;
    }

    public function getIdclient(): ?int
    {
        return $this->idclient;
    }

    public function setIdclient(?int $idclient): static
    {
        $this->idclient = $idclient;

        return $this;
    }

    public function getLieu(): ?string
    {
        return $this->lieu;
    }

    public function setLieu(?string $lieu): static
    {
        $this->lieu = $lieu;

        return $this;
    }

    public function getEtat(): ?string
    {
        return $this->etat;
    }

    public function setEtat(?string $etat): static
    {
        $this->etat = $etat;

        return $this;
    }


}
